data-science-pipelines/overlays/component-visualization-server/deployments/ds-pipeline-visualizationserver.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   annotations:
     alpha.image.policy.openshift.io/resolve-names: 'visualization-server:1.3.1'
     image.openshift.io/triggers: >-
-      [{"from":{"kind":"ImageStreamTag","name":"visualization-server:1.3.1"},"fieldPath":"spec.template.spec.containers[?(@.name==\"ds-pipeline-visualizationserver\")].image"}]
+      [{"from":{"kind":"ImageStreamTag","name":"visualization-server:1.3.1"},"fieldPath":"spec.template.spec.containers[?(@.name==\"ml-pipeline-visualizationserver\")].image"}]
```

Comment:
@rimolive Should this container name stay `ds-pipeline-visualizationserver`?

Reply:
There were no code changes, just files moved from one location to another. This shouldn't be an issue as these manifests were tested before.

Reply:
Gotcha.